styles/utility.css

```diff
@@ -473,6 +481,119 @@ textarea {
   color: #344563;
 }
 
+.modal {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.1);
+  z-index: 2;
+  animation: modalopen 1s ease-out;
+  display: none;
+}
+
+.modal-container {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 60px 16px 0 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  min-width: 343px;
+  height: 100%;
+  padding: 18px 16px;
+}
+
+.modal-wrapper {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 18px;
+  width: 378px;
+  height: 132vh;
+  margin: 0 auto;
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+}
+
+.project-image {
+  margin-top: 20px;
+}
+
+.close i {
+  display: flex;
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+  margin-right: -50px;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.modal-project-desc-tech-details {
+  margin-top: 15px;
+  text-align: left;
+}
+
+.project-title {
+  font-size: 32px;
+  font-weight: 600;
+  color: #344563;
+  padding-left: 20px;
+  line-height: 32px;
+}
+
+.tech-list-modal-container {
+  display: flex;
+  width: 100%;
+  margin: 12px 20px 0 0;
+  justify-items: normal;
+  justify-content: space-evenly;
+  flex-flow: row nowrap;
+  align-items: center;
+}
+
+li.project-technologies {
+  border-radius: 8px;
+  background: #ebf0ee;
+  padding: 12px;
+}
+
+li.project-technologies p {
+  color: #3a4a42;
+  font-weight: 600;
+  font-size: 12px;
+}
+
+p.modal-text-content {
+  margin: 20px 12px 12px;
+  line-height: 20px;
+}
+
+.modal-buttons .btn {
+  display: inline-flex;
+  flex-direction: row;
+  width: 45%;
+  justify-content: space-between;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  background-color: #36b37e;
+  color: #fff;
+  font-size: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border-radius: 4px;
+  align-self: center;
+}
```

Comment:
Please remember that a responsive design must work in all screen sizes, in this case, your design looks great, but it's not centered on the desktop version as you can see in the following screenshots:

Your project:
![image](https://user-images.githubusercontent.com/82678907/152261020-48a4e894-2582-4c21-a2fe-b1cb84d40860.png)

Figma Design:
![image](https://user-images.githubusercontent.com/82678907/152261228-1f9d8465-8dd4-4460-9501-cfaad371cb76.png)

- [ ] Kindly adjust your design to Figma design in the desktop version.

Reply:
I have made the desktop view to be just as it is in the design